content/03.aim1.md

```diff
@@ -124,3 +124,29 @@ The performance of VAE is generally better than the other approaches for majorit
 ![](images/real.data.knn.png)
 
 Figure 7: Performance comparison VAE, t-SNE, ZIFA, UMAP and PCA based on k-nearest neighbor algorithm.
+
+#### Data augmentation
+
+Data augmentation is a way to reduce overfitting on models by increasing the amount of training data using information only in the current training set. 
+In images, data augmentation can be obtained by arbitrary rotations, zooms, and other irrelevant transformations of images to increase the amount of training data, which allows a deep learning model to capture the differences from the data and reduces the model's propensity to learn noise. 
+The same concept also applies to single-cell transcriptomes. 
+We hypothesized that genomic data augmentation will improve latent feature generalization by separating biological from technical features and increasing the effective sample size during training.
+
+We performed data augmentation based on HCA melanoma dataset. 
+We down sampled the melanoma dataset to test the method stability.
+The dataset was bootstrapped with 100, 500, 1000 and 2000 cells. 
+We expected data augmentation will contribute more for small number of cells rather than large number of cells.
+Salmon (version 0.8.1) [@doi:10.1038/nmeth.4197] was used to do the transcript abundance quantification and we resampled the reads by 2x, 5x and 10x times to capture the uncertainty in the data. 
+
+Figure 7 and 8 show the performance of data augmentation under different conditions. 
+K-means and Knn based performance both indicate data augmentation works well in small number of samples (100 and 500 cells), but does not contribute too much in a large sample size (2000).
```

Comment:
Maybe: "... works well for very small numbers of cells (100), but does not appear to help when the number of cells is modest (2000).